bench/micro/fiber_bench.ml

```diff
@@ -27,21 +27,29 @@ let%bench_fun ("ivar" [@indexed jobs = [ 100; 10_000; 1_000_000 ]]) =
      in
      loop jobs)
 
-let%bench_fun ("ivar" [@indexed jobs = [ 100; 10_000; 1_000_000 ]]) =
- fun () ->
-  let ivar = ref (Fiber.Ivar.create ()) in
-  Fiber.run
-    ~iter:(fun () ->
-      let open Nonempty_list in
-      [ Fiber.Fill (!ivar, ()) ])
-    (let rec loop = function
-       | 0 -> Fiber.return ()
-       | n ->
-         ivar := Fiber.Ivar.create ();
-         let* () = Fiber.Ivar.read !ivar in
-         loop (n - 1)
-     in
-     loop jobs)
+let%bench_fun ("Var.set" [@indexed jobs = [ 100; 10_000; 1_000_000 ]]) =
```

Comment:
For var, I think it's enough to jobs = 1000 (or whatever value). Probably same for the ivar benchmarks as well. I don't see any non linearity arising here.

Reply:
Sounds about right. In fact, for `bind` as well.

Reply:
For bind, I just want to make sure we aren't leaking memory by waiting until the last bind to release everything. I'm pretty sure fiber doesn't suffer from this problem though :)

Reply:
I did that

Reply:
Ah, I see. We have tests for this kind of stuff in the fiber expect tests,  so we could add some more tests for this.